collector/kernel/cgroup_handler.h

```diff
@@ -49,7 +47,7 @@ class CgroupHandler {
   void handle_pid_info(u32 pid, u64 cgroup, uint8_t comm[16]);
 
 private:
-  FRIEND_TEST(cgroup_handler_test, HandleDockerResponse);
+  friend class cgroup_handler_test_HandleDockerResponse_Test;
```

Comment:
Is this the kind of thing where we want to have the code in ifdef blocks to prevent integration into production code?